mule-module-apikit/src/test/resources/org/mule/module/apikit/proxy/app-with-raml/mule-config.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:http="http://www.mulesoft.org/schema/mule/http"
+      xmlns:apikit="http://www.mulesoft.org/schema/mule/apikit"
+      xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+        http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+        http://www.mulesoft.org/schema/mule/apikit http://www.mulesoft.org/schema/mule/apikit/current/mule-apikit.xsd">
+
+    <apikit:proxy-config name="proxy-config" raml="proxy.raml" disableValidations="true"/>
+
+    <flow name="proxy">
+        <http:inbound-endpoint port="${proxyPort}" host="localhost" path="proxy"/>
```

Comment:
could you use `listener` and `request` instead of inbound and outbound endpoints?